AGS02MA.h

```diff
@@ -28,6 +28,11 @@
 class AGS02MA
 {
 public:
+  struct ZeroCalibration {
```

Comment:
should it be called ZeroCalibrationData  to prevent confusion with the function with same name?
(no showstopper ;)

Reply:
> Do we have information about the different values status can have ? what they mean etc?
Need to mention "warning: this is experimental / under investigation" ?

Not really. Only these observations so far.

```
0000 1100 | 0x0C | 12
0000 1101 | 0x0D | 13
0111 1101 | 0x7D | 125
```

- It looks like a bitmask.
- `12` is the value v117 seems to have normally. v118 has this *immediately* after setting calibration.
- `13` I've only seen in your v117 dumps. I don't know under which conditions. I've not seen it on v118.
- `125` I've only seen on v118, after power-cycling it. Typically this means you get different PPB values from `12`!

Reply:
With mentioning a warning, do you mean adding something like this?

```cpp
  struct ZeroCalibrationData
  {
    /**
     * Warning, the exact meaning of this status is not fully undocumented.
     * It seems like it's a bitmask:
     * 0000 1100 | 0x0C | 12  | Typical value
     * 0000 1101 | 0x0D | 13  | Sometimes seen on v117
     * 0111 1101 | 0x7D | 125 | Seen on v118, after power-off (gives different data than "12")
     */
    uint16_t status;
    uint16_t value;
  };
```

Reply:
I've included this warning in source now. Assuming that's what you meant.